assets/Icons.tsx

```diff
@@ -0,0 +1,33 @@
+export const TrashCan = () => (
```

Comment:
These are React components and should be under `components`, in a file like `components/icons.tsx`.